### DISCUSSION TITLE
Which IP Address we should choose?

### DISCUSSION THREAD

**Discussion by vareddy101091** [learndevopsonline/learndevopsonline#504] [Q&A] [answered]
Hi Raghu,

i have observed you have choosen public IP Address from some instances and private IP Address from some instances to run the microservices can you please explain which ip address we should select generally?

thank you.

**Comment by Manju9876** [comment, marked as answer]
hi @vareddy101091, 
public IP address is only used to open the virtual machine through putty and only frontend  public IP is used to browse the display of content of the project.
where as in private IP addresses we use them to  link every component's in the project  since its is micro service , private IP is used to interact with other components in the project.
